### PR TITLE
store/tikv: fix panic

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @disksing PTAL

File: store/tikv/coprocessor.go [hunk 1]
```diff
@@ -392,6 +392,9 @@ func (it *copIterator) Next() (io.ReadCloser, error) {
 	if it.mu.respGot == len(it.mu.tasks) {
 		it.mu.finished = true
 	}
+	if resp == nil {
```
Comment (shenli) [inline store/tikv/coprocessor.go]: Check it.mu.finished == true?

**Comment by shenli** [comment]
LGTM

**Comment by c4pt0r** [comment]
panic:

2016/09/08 06:15:12 coprocessor.go:437: [warning] coprocessor region error: message:"peer is not leader" not_leader:<region_id:281 > , retry later
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x15de30]

goroutine 256772 [running]:
panic(0x934948, 0x11324008)
          /home/pi/go/src/runtime/panic.go:500 +0x33c
github.com/pingcap/tidb/store/tikv.(*copIterator).Next(0x154c17c0, 0x0, 0x0, 0x0, 0x0)
          /home/pi/gopkg/src/github.com/pingcap/tidb/store/tikv/coprocessor.go:390 +0x280
github.com/pingcap/tidb/xapi.(*selectResult).fetch(0x1612ede0)
          /home/pi/gopkg/src/github.com/pingcap/tidb/xapi/xapi.go:85 +0x70
created by github.com/pingcap/tidb/xapi.(*selectResult).Fetch
          /home/pi/gopkg/src/github.com/pingcap/tidb/xapi/xapi.go:79 +0x2c

**Comment by coocood** [comment]
@disksing PTAL

**Comment by disksing** [comment]
LGTM

**Pull request merged**